### ISSUE TITLE
SEEK_cHostEntryFileInformation - add required constraint to HostIpAddress

### ISSUE THREAD

**Issue by xelibrion** [SEEK-Jobs/DSC#28]
Currently it is possible to specify empty value for HostIpAddress, which leads to unexpected behaviour when applying configuration.

**Comment by warnergodfrey** [comment]
Should be pretty simple to add some validation, do you want to have a go?

**Comment by xelibrion** [comment]
@warnergodfrey I wouldn't mind, but it looks like `SEEK_cHostEntryFileInformation ` should be a resource on its own - entries in hosts file shouldn't be tied to IIS bindings, as there is an option not to use IIS at all.

**Comment by warnergodfrey** [comment]
True. I was hoping to extract that behaviour in v2. Let me have the first cut.



On 5 May 2015, at 5:55 pm, Dmitry Kryuchkov <notifications@github.com<mailto:notifications@github.com>> wrote:


@warnergodfrey<https://github.com/warnergodfrey> I wouldn't mind, but it looks like SEEK_cHostEntryFileInformation should be a resource on its own - entries in hosts file shouldn't be tied to IIS bindings, as there is an option not to use IIS at all.
